core/src/main/resources/jenkins/slaves/RemotingWorkDirSettings/help-failIfWorkDirIsMissing.html

```diff
@@ -0,0 +1,4 @@
+<div>
+  If defined, Remoting will fail the startup if the target work directory is missing.
```

Comment:
"Remoting will fail at startup" I think rather than "Remoting will fail the startup"